02-peer-protocol.md

```diff
@@ -938,6 +960,38 @@ previously sent.
 A receiving node MAY ignore spurious message retransmission, or MAY
 fail the channel if they occur.
 
+### Rationale
+
+The effect of requirements above are that the opening phase is almost
+atomic: if it doesn't complete, it starts again.  The only exception
+is where the `funding_signed` message is sent and not received: in
+this case, the funder will forget the channel and presumably open
+a new one on reconnect; the other node will eventually forget the
+original channel due to never receiving `funding_locked`.
```

Comment:
`push_msat` was never intended to be a trusted workaround to set up balanced channels. It simply attempts to address the case wherein the channel initiator wishes to _immediately_ pay the responder.